### DISCUSSION TITLE
What distribution curve is best for miner incentives and validator vTrust?

### DISCUSSION THREAD

**Discussion by sangar-1028** [web-genie-ai/web-genie-ai#8] [Q&A] [answered]
For miner incentives, we should have a strong incentive curve where miner rewards are distributed effectively. 
However, vTrust is supposed to rely on this distribution curve.
What curve would be ideal?

**Comment by mrbreo** [comment, marked as answer]
The S-curve distribution would be the best option. In fact, we can utilize several curves, such as the Sigmoid function or cubic function.

**Comment by donbusha** [comment]
Alright, S-curve distribution works well, and we can choose one of these functions.
